### PR TITLE
Add color management for Windows

### PR DESCRIPTION
**Pull request by wwww-wwww**

### CODE REVIEW

File: vspreview/core/types/video.py [hunk 1]
```diff
@@ -172,10 +172,10 @@ class ChromaLoc:
         'frame_to_show', 'scening_lists'
     )
     __slots__ = storable_attrs + (
-        'index', 'width', 'height', 'fps_num', 'fps_den',
-        'total_frames', 'total_time', 'graphics_scene_item',
```
Comment (cid-chan) [inline vspreview/core/types/video.py:176]: Please, do try to keep the changes to a minimum.
Especially for reformatting of unrelated stuff.

**Review by cid-chan — COMMENTED** [review]
Also, be aware that my attempts on yuuno to introduce color management ended in the request to disable it.

I don't think it will be well received.

**Comment by wwww-wwww** [comment]
Totally understandable. I'm trying to make it least disruptive and I'm hoping it can at least be put in being disabled by default.
It obviously runs slower with it on but it's part of a price I and probably a lot of others are willing to pay for correct colors.

File: vspreview/main/settings.py [hunk 2]
```diff
@@ -131,6 +131,10 @@ def setup_ui(self) -> None:
             self.timeline_notches_margin_spinbox
         )
 
+        self.color_management_checkbox = QCheckBox(self)
+        self.color_management_checkbox.setText('Color management')
+        self.vlayout.addWidget(self.color_management_checkbox)
```
Comment (cid-chan) [inline vspreview/main/settings.py:136]: Oh, I missed this: But if you only support color management on Win32, might as well hide the checkbox on other platforms.

**Review by cid-chan — DISMISSED** [review]

**Comment by YomikoR** [comment]
I'm not sure whether working in 8-bit without dithering is satisfactory. Would you consider taking the transform in RGBX64? That is, resizing to vs.RGB48, packing to RGBA64, having color management in RGBX64, and finally dithered by Qt with error diffusion.

**Comment by cid-chan** [comment]
Are you still working on this?

**Comment by wwww-wwww** [comment]
Since there's a plan to switch to pyside this will all probably be not used then, so I'm hesitant in bothering to keep this up to date.
